lib/src/presentation/view/service_request/widgets/service_grid_tile.dart

```diff
@@ -1,6 +1,5 @@
 
 import 'package:coofix/src/application/get_servieces/get_services_bloc.dart';
-import 'package:coofix/src/application/get_servieces/get_services_state.dart';
 import 'package:coofix/src/presentation/core/constants/images.dart';
 import 'package:coofix/src/presentation/core/constants/strings.dart';
 import 'package:coofix/src/presentation/core/theme/colors.dart';
```

Comment:
In here also item counts are hard coded. Please make sure the flow and purpose of the project

Reply:
finish